pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotAdministrator.java

```diff
@@ -190,13 +190,9 @@ public static void main(String[] args) {
     PluginManager.get().init();
     PinotAdministrator pinotAdministrator = new PinotAdministrator();
     pinotAdministrator.execute(args);
-    // Ignore `pinot.admin.system.exit` property for Pinot quickstarts.
-    if ((args.length > 0) && ("quickstart".equalsIgnoreCase(args[0]))) {
-      return;
-    }
-    if (Boolean.parseBoolean(System.getProperties().getProperty("pinot.admin.system.exit"))) {
-      // If status is true, cmd was successfully, so return 0 from process.
-      System.exit(pinotAdministrator._status);
+    if (Boolean.parseBoolean(System.getProperties().getProperty("pinot.admin.system.exit"))
```

Comment:
Should we add a switch instead of env variable?

Reply:
We can definitely add, but for backward compatible, we still need to keep this.